src/main/java/seedu/address/logic/commands/AddVisitCommand.java

```diff
@@ -28,33 +31,51 @@
             + "Date (must be in the format of yyyy-MM-dd "
             + "Example: " + COMMAND_WORD + " 1 " + " 2 " + PREFIX_DATE + " 2020-09-09 ";
 
-    private final Visit toAdd;
+    private final Index personIndex;
+    private final Index locationIndex;
+    private final LocalDate date;
 
     /**
      * Creates an AddVisitCommand to add the specified {@code Visit}
      */
-
-    public AddVisitCommand(Visit visit) {
-        requireAllNonNull(visit);
-        toAdd = visit;
+    public AddVisitCommand(Index personIndex, Index locationIndex, LocalDate date) {
+        requireAllNonNull(personIndex, locationIndex);
```

Comment:
Shouldn't date be required to be non null as well?

Reply:
Thanks for the catch! I have updated it.